src/lib.rs

```diff
@@ -1200,6 +1206,7 @@ impl<'a, T: 'a> ExactSizeIterator for SliceMutIter<'a, T> {}
 
 /// Returns the highest power of 2 that is less than or equal to `v` when `v` is non-zero.
 /// If `v` is zero, `None` is returned.
+#[cfg(test)]
 fn checked_log2_floor(v: usize) -> Option<u32> {
```

Comment:
If this is no longer used by the implementation, it is okay to remove it and the associated test.